azure/packages/azure-client/package.json

```diff
@@ -22,7 +22,7 @@
 		"build:esnext": "tsc --project ./tsconfig.esnext.json",
 		"build:test": "tsc --project ./src/test/tsconfig.json",
 		"ci:build:docs": "api-extractor run",
-		"clean": "rimraf dist lib *.tsbuildinfo *.build.log",
+		"clean": "rimraf --glob dist lib *.tsbuildinfo *.build.log",
```

Comment:
I believe before this change, this command worked on some OSes (like linux) because the shell was expanding the "*". It seems like if we are going to use --glob, for consistency it would be better to have rimraf expand it on all platforms, so we should quote the patterns to stop the shell from doing it.